chart/values.yaml

```diff
@@ -315,6 +315,7 @@ varnish:
 elasticsearch:
   enabled: false
   replicas: 1
+  minimumMasterNodes: 1
```

Comment:
This doesn't belong in here :-)

Reply:
yep, noticed the difference between chart and k8s, decided to bring it along. it's a separate commit :)